examples/collector/src/Makefile

```diff
@@ -0,0 +1,69 @@
+# http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+PLUGINTYPE=collector
+NAME=rando
+TARGET=test-$(PLUGINTYPE)-$(NAME)
+
+SNAPLIBS=/usr/local/lib
+ICCROOT=
+EROOT=../..
+BDIR=$(EROOT)/bin
+
+CC=g++
+IDIRS=-I.
+LDIRS=-L.
+CFLAGS=-std=c++1y
+LIBS=-Wl,--start-group $(SNAPLIBS)/libsnap.a $(SNAPLIBS)/libgrpc.a $(SNAPLIBS)/libgrpc++.a $(SNAPLIBS)/libprotobuf.a -Wl,--end-group -lz -lpthread
```

Comment:
We can add this package requirement to the .travis.yml when we start building the plugins.